torch/_inductor/overrides.py

```diff
@@ -110,22 +110,23 @@ def fetch_attr(target: str, mod):
     return attr_itr
 
 
+# Note - This is a CPU only pass, see fuse_fx
 def remove_identity(gm: torch.fx.GraphModule):
     """
     Removes all identity layers from the module.
     """
 
     class IdentityRemover(torch.fx.Transformer):
         def call_module(self, target, args, kwargs):
-            if isinstance(self.submodules[target], nn.Identity):
+            if isinstance(self.module._modules[target], nn.Identity):
```

Comment:
This looks like trouble, did you make a BC breaking change?

Reply:
I do not think so! For some reason, the name is not in self.submodules, but it is in self.module._modules! I need to look at this closer.

Reply:
I think I know what this is though, let me double check...